src/main/java/com/omega/Proyecto/omega/Repository/IRepositorySale.java

```diff
@@ -2,8 +2,23 @@
 
 import com.omega.Proyecto.omega.Model.Sale;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface IRepositorySale extends JpaRepository<Sale, Long> {
+    @Query("SELECT S FROM Sale S WHERE S.active = true")
+    List<Sale> getActivaSale();
```

Comment:
```suggestion
    @Query("SELECT S FROM Sale S WHERE S.active = true")
    List<Sale> getActiveSale();
```
se coló una errata en la función no me pegues jaja
Solo por comprobar... a veces si se usan en jpa nombres concisos no hacen falta definir la query, has probado a añadir el método 
`List<Sale> getSalesByActive(@Param("active") boolean active);`